app/interactors/cangaroo/perform_jobs.rb

```diff
@@ -15,7 +16,18 @@ def data
     end
 
     def enqueue_jobs(type, payload)
-      initialize_jobs(type, payload).select(&:perform?).each(&:enqueue)
+      initialize_jobs(type, payload).each do |job|
+        if job.perform?
+          # TODO log simplified info about payload info is being queued
+          job.enqueue
+        else
+          # TODO persist translation for audit trail?
```

Comment:
This is referring to the `Translation` record on this PR https://github.com/nebulab/cangaroo/pull/19

If the job is not queued it won't be saved to the translation log. However, I think it would be helpful to have a record of the translation in the log.